src/Microsoft.AspNet.Server.Kestrel/Http/FrameRequestStream.cs

```diff
@@ -165,7 +165,26 @@ private void ValidateState()
                 case FrameStreamState.Closed:
                     throw new ObjectDisposedException(nameof(FrameRequestStream));
                 case FrameStreamState.Aborted:
-                    throw new IOException("The request has been aborted.");
+                    throw new OperationCanceledException("The request has been aborted.");
```

Comment:
After having crazy exception handling/checking for Websockets in 4.5+ (I think SignalR had similar) - I'm all up for a standardised exception :)